src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -151,6 +151,10 @@ internal static IEnumerable<ExtendedTypeDefinition> GetFormatData()
                 "System.Management.Automation.ScriptBlock",
                 ViewsOf_System_Management_Automation_ScriptBlock());
 
+            yield return new ExtendedTypeDefinition(
```

Comment:
Should this check if the experimental feature is enabled?

Reply:
This won't have any functional impact if there's a formatter and that type is never created without the cmdlet.